scripts/rename_docker_images.py

```
@@ -60,6 +72,14 @@ def get_current_timestamp() -> str:
     return _dt.datetime.now(_dt.timezone.utc).strftime("%Y%m%d%H%M")
 
 
+def get_current_commit() -> str:
+    """Return the 7-char short SHA of the current git HEAD."""
+    return _sp.run(["git", "rev-parse", "--short=7", "HEAD"],
```

Ruff (0.15.14)

[error] Starting a process with a partial executable path

(S607)